enhancements/observability/multicluster-observability-addon-interoperability.md

```diff
@@ -0,0 +1,217 @@
+uses three different technology stacks and protocols for exporting data (Prometheus 
+remote-write for metrics, Loki push for logs and OTLP for traces). 
+Loki push and Prometheus remote-write are not commonly supported as ingest protocols by 
+observability vendors.
+
+This enhancement proposal seeks to strengthen interoperability of MCOA by unifying and 
+simplifying exporting of all MCOA telemetry data (metrics, logs, traces) 
+by exposing a unified export API and consolidating export protocols. This capability
+enables users to send data from MCOA to any observability vendor and apply 
+fine-grained filtering and routing on exported data to configurable sinks.
+
+## Motivation
+
+At the moment exporting all telemetry data from OpenShift is fragmented to tree 
+technology stacks (Prometheus, ClusterLoggingForwarder, OpenTelemetry collector). 
+Every tool uses a different configuration API, export protocol and provides (or does 
+not provide) filtering/PII capabilities.
+
+### Prior art and user requests
+
+* Red Hat OpenShift as OpenTelemetry (OTLP) native platform: https://www.redhat.com/en/blog/red-hat-openshift-opentelemetry-otlp-native-platform
+* Export in-cluster metrics to 3rd party vendor https://github.com/openshift/cluster-monitoring-operator/issues/2000
+* Exporting metrics to Dynatrace https://issues.redhat.com/browse/OBSDA-433
+* Export all metrics to Dynatrace https://issues.redhat.com/browse/OBSDA-450
+* Customer asking to export metric to Splunk https://redhat-internal.slack.com/archives/C04TFRRKUA2/p1687853284985279
+
+### User Stories
+
+* As a fleet administrator, I want to export all telemetry signals collected by MCOA to an OTLP compatible endpoint(s).
+* As a fleet administrator, I want to filter sensitive data before it is exported to MCO telemetry store or 3rd party OTLP endpoint.
+* As a fleet administrator, I want to decide which data is exported to MCO telemetry store or 3rd party OTLP endpoint.
+
+### Goals
+
+* Use a single protocol (OTLP) for exporting all data to MCO and/or 3rd party system.
+* Provide a single configuration API on MCO CRD for exporting all telemetry data.
+* Provide unified filtering and routing capabilities for all exported telemetry data.
+
+### Non-Goals
+
+* Data visualization and querying.
+
+## Proposal
+
+The following sections describe in detail the required resources as well as the workflow to enable collection and forwarding of logs and traces on an RHACM managed fleet of OCP clusters.
+
+The following section describes how data exportation, routing and filtering is 
+configured in MCO and MCOA.
+
+![Architecture](./multicluster-observability-addon-interoperability-arch.jpg)
+
+### Workflow Description
+
+1. Configure OTLP endpoint in MCO (`MultiClusterObservability`) CR.
+2. The MCOA configures an additional OTLP exporter in the OpenTelemetry collector. The 
+   exporter is in the pipeline that receives all data.
+3. (optional) Filtering (e.g. for PII) can be configured in 
```

Comment:
> Is the expectation that users will configure their own PII filtering? 

yes, and most likely mostly for user workload metrics. They could filter out platform data as well, but it will be their responsibility if they break the console. 

logs are traces are more important for PII than metrics.

> I'm not clear as to how this would be easier/unified, Such filtering can be configured on prometheus scrape-level or for ClusterLoggingForwarder as well and actually having them split ensures a user can be intentional about what exactly they want to filter.

With MCO we are intentional about making it easy to provision and manage the entire stack and ultimately provide a good integrated product experience. As a user I would prefer to configure processing/filtering capability in a single API rather than on three different APIs/stacks (they could even have different processing/filtering capabilities).